noxfile.py

```diff
@@ -30,6 +30,9 @@
 
 BLACK_VERSION = "black==22.3.0"
 ISORT_VERSION = "isort==5.12.0"
+
+# pytest-retry is not yet compatible with python 8.x.
```

Comment:
```suggestion
# pytest-retry is not yet compatible with IPython 8.x.
```

Reply:
Please could you add a link to the upstream issue if it exists?

Reply:
Filed and linked here, thanks: https://github.com/str0zzapreti/pytest-retry/issues/32